setup.py

```diff
@@ -71,7 +71,7 @@ def run_tests(self):
         'pylmod',
         'PyYAML',
         'uwsgi',
-        'WTForms',
+        'Flask-WTF',
     ],
```

Comment:
Flask-WTF has a dep on WTForms, so is no longer needed, and best handled by letting Flask-WTF determine the version it wants.